passport-reshuffle/src/strategy.ts

```diff
@@ -0,0 +1,76 @@
+import { Profile, Strategy as StrategyType } from 'passport';
+import { Strategy as LocalStrategy } from 'passport-local';
+import { Strategy as Auth0Strategy } from 'passport-auth0';
+import { titlecase } from 'stringcase';
+import { stickFigure } from './stick-figure';
+
+// Currently the *base* Passport Profile.  But could be anything
+// supported by the underlying Passport strategy.
+export { Profile };
+
+function getMissingOauthEnvariables() {
+  return ['OAUTH_DOMAIN', 'OAUTH_CLIENT_ID', 'OAUTH_CLIENT_SECRET']
```

Comment:
what about `RESHUFFLE_SESSION_SECRET`?

Reply:
It explicitly works without for local mode.